assets/app/styles/_sidebar.less

```diff
@@ -192,8 +191,14 @@
               }
               ul.nav-sidenav-secondary {
                 margin-bottom: 20px;
-                > li > a {
-                  padding: 10px 0;
+                > li {
+                  &.active a {
```

Comment:
`&.active > a`? not sure why immediate-child selectors were there for the other anchor rule here